website/docs/r/satellite_location.html.markdown

```diff
@@ -85,6 +87,8 @@ Review the argument references that you can specify for your resource.
   Nested scheme for `cos_credentials`:
   - `access_key-id` - (Required, String)The `HMAC` secret access key ID.
   - `secret_access_key`-  (Optional, String) The `HMAC` secret access key.
+- `physical_address` -  (Optional, String) The physical address of the new Satellite location which is deployed on premise.
+- `capabilities` - (Optional, Array of Strings) Satellite capabilities attached to the location. It is mandatory to add the value 'on-prem' to 'capabilitiesManagedBySatellite', if a value has been set for 'physicalAddress'. On the other hand, value can be added to 'capabilitiesManagedBySatellite' without setting any value to 'physicalAddress'. In other words, 'capabilitiesManagedBySatellite' is optional, unless 'physicalAddress' gets set.
```

Comment:
This description is n't clear what do you mean `capabilitiesManagedBySatellite` here 
Can you check if you can add any `RequiredWith` at schema level

Reply:
This is because, in Kubernetes API, capabilities is called capabilitiesManagedBySatellite. Now, I also added the [API link ](https://cloud.ibm.com/apidocs/kubernetes/containers-v1-v2#createsatellitelocation) with this info to the markdown.
Thanks for the RequiredWith tipp. Also, added RequiredWith-physical_address to capabilities in resource code.